wikiweaver-ext/content/join-lobby.js

```diff
@@ -0,0 +1,31 @@
+const fragment = document.createDocumentFragment();
+
+const div = document.createElement("div");
+div.classList.add("flex-horizontal-container")
+
+// TODO: There is probably a better way to do all this
+const btn = document.createElement("btn");
+btn.id = "join-lobby";
+btn.classList.add("button", "box", "text");
+btn.onclick = HandleJoinLobbyClicked;
+btn.style = "margin-top: 0.5rem; background: var(--green);";
+btn.textContent = "join";
```

Comment:
i think you can put the div and button in the html and make the div `hidden`: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden